sdks/python/apache_beam/pipeline_test.py

```diff
@@ -185,65 +180,6 @@ def apply(self, pcoll):
         ['a-x', 'b-x', 'c-x'],
         sorted(['a', 'b', 'c'] | 'AddSuffix' >> AddSuffix('-x')))
 
-  def test_cached_pvalues_are_refcounted(self):
```

Comment:
Create a PCollection with a dozen 1MB strings. Now apply 100 Map operations that append a character to each element. Total memory increase should stay in the MBs, not GBs.